test/testdata

```diff
@@ -155,7 +164,6 @@ MP        QChem       QChemMP4SDQTest       basicQChem4.2         water_mp4sdq.o
 MP        QChem       QChemMP4SDTQTest      basicQChem4.2         water_mp4.out
 
 Polar     DALTON      ReferencePolarTest    basicDALTON-2015      Trp_polar_abalnr.out
-Polar     GAMESS      ReferencePolarTest    basicGAMESS-US2012    Trp_polar_tdhf.out
```

Comment:
Done. I re-ran the calculation with 2017 and removed the basicGAMESS-2012 folder.